[IMP] sales: subscriptions follow up alerts add iap (#8822)
odoo/documentation: content/applications/sales/subscriptions/follow_up/automatic_alerts.rst

      *Trigger date* and *Delay after trigger* need to be specified. And because the *Send an email
      to the customer* action was adopted, an *Email template* can be chosen.

   .. note::

>       Sending a SMS test message in Odoo requires In-App Purchase (IAP) credit or tokens. For more

Reviewer: ```suggestion
      Sending a SMS text message in Odoo requires In-App Purchase (IAP) credit or tokens. For more
```

>       information on :abbr:`IAP (In-App Purchase)` visit

Reviewer 2: ```suggestion
      information on :abbr:`IAP (In-App Purchase)`, visit
```

>       :doc:`../../../essentials/in_app_purchase`. For more information on sending SMS messages visit

Reviewer 2: ```suggestion
      :doc:`../../../essentials/in_app_purchase`. For more information on sending SMS messages, visit
```

      visit :doc:`../../../marketing/sms_marketing/essentials/sms_essentials`.

> As a result, this alert will send a rating survey after one month, to the customers who have

Reviewer: ```suggestion
As a result, this alert sends a rating survey after one month, to the customers who have
```

> purchased that specific product. The survey will appear in the chatter of your respective

Reviewer: ```suggestion
purchased that specific product. The survey appears in the chatter of your respective
```
